src/qt/bitcoingui.cpp

```diff
@@ -928,8 +944,12 @@ void BitcoinGUI::NewUserWizard()
                 QString qsMessage = tr(sBoincNarr.c_str());
                 QMessageBox::warning(this, tr("Attention! - Boinc Path Error!"), qsMessage);
         }
-
-
+    }
+    else
+    {
+        QString strMessage = tr("Note: gridcoinresearch.conf is not empty. If you want to run the new user wizard again, shutdown the wallet, delete the conf file and restart.");
```

Comment:
This pops up every time someone runs the wallet if a config file exists...seems a bit much.

Reply:
I don’t think so because the event is never called to invoke the function.

Reply:
You may be right. The original just silently exited if the config file is non-empty.

Reply:
You may be right. The original just silently exited if the config file is non-empty.

Reply:
I think we can move this message so that it shows if the user clicks "cancel". That way, someone still gets instructions for running the wizard again, but it doesn't pop up a window each launch for most people that don't need to see it.

Reply:
For right now I just removed the else statement. We can fix this when we overhaul the whole thing.